packages/components/src/index.js

```diff
@@ -67,6 +67,7 @@ export { default as MenuItemsChoice } from './menu-items-choice';
 export { default as Modal } from './modal';
 export { default as ScrollLock } from './scroll-lock';
 export { NavigableMenu, TabbableContainer } from './navigable-container';
+export { default as __experimentalNavigation } from './navigation';
```

Comment:
Should we also expose `NavigationMenuItem` from here as well?

Reply:
Oh right! Added in `NavigationMenu` as well in https://github.com/WordPress/gutenberg/pull/24706/commits/bdf9ed8f85ce9de7e3e2438c9e55453324a88d51